app/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@zeit/next-css": "^1.0.1",
     "express": "^4.17.0",
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
Is there any specific reason to use this dependency instead of the generic fetching?

Reply:
The isomorphic version works properly with the client and backend side (SSR)